bundle/tests/path_translation/nominal/override_job.yml

```diff
@@ -0,0 +1,41 @@
+targets:
+  development:
+    resources:
+      jobs:
```

Comment:
Just to confirm, is this test case failed before the change? The reason I ask is that I remember the failed use case was "a job defined in root and override is in subfolder". I'm not sure if it matters in this case

Reply:
Indeed. The fallback one passed and this one failed. I'll manually confirm the case where the reverse setup is used like you mention, but it shouldn't matter.